webroot/js/toolbar.js

```diff
@@ -29,7 +37,13 @@
 		}
 		var body = doc.body;
 		iframe = doc.createElement('iframe');
-		iframe.setAttribute('style', 'position: fixed; bottom: 0; right: 0; border: 0; outline: 0; overflow: hidden; z-index: 99999;');
+        iframe.style.position = 'fixed';
+        iframe.style.bottom = 0;
+        iframe.style.right = 0;
+        iframe.style.border = 0;
+        iframe.style.outline = 0;
+        iframe.style.overflow = 'hidden';
+        iframe.style.zIndex = 99999;
```

Comment:
@dakota Well I won't say that CSP is breaking things, but you have to do some changes, which could be a bit tough work to do. But it will bring much protection against XSS attacks if configured right. :wink: 
For example I also won't say 64 bit support for a program will break more things than it fixes, but you have to do much work to change the program to support 64 bit. :wink: